x-pack/plugin/data-frame/qa/single-node-tests/src/test/java/org/elasticsearch/xpack/dataframe/integration/DataFrameTaskFailedStateIT.java

```diff
@@ -30,53 +29,47 @@ public void testFailureStateInteraction() throws Exception {
         createReviewsIndex();
         String transformId = "failure_pivot_1";
         String dataFrameIndex = "failure_pivot_reviews";
-        createPivotReviewsTransform(transformId, dataFrameIndex, null);
-        deleteIndex(REVIEWS_INDEX_NAME); // trigger start failure due to index missing
+        createContinuousPivotReviewsTransform(transformId, dataFrameIndex, null);
         startDataframeTransform(transformId, false);
+        // wait for our initial indexing to complete
```

Comment:
These test changes are in here as I was able to get the test to pass with other code changes. However, those other changes may not need to exist within 7.3. Keeping the test changes for future help when we enable this test again but keeping it muted for now.

Reply:
I don't think this test is necessary. It goes to great lengths to set up a transform that can fail when all we really want to do is test that if the cluster state has failed data frame persistent tasks an error and 409 is returned.